cheatsheets/cors.md

```diff
@@ -7,6 +7,17 @@ Check to see what the server responds with in the `Access-Control-Allow-Origin:`
 
 If it is trusting arbitrary origins **with** allow-credentials set to true, then host this HTML as a proof of concept.
 
+## Some method to bypass CORS	
```

Comment:
I'd like to suggest that the title be changed into "Origin-based CORS bypass", to make it more professional